src/components/SiteHeader.vue

```diff
@@ -32,11 +32,13 @@
             <GlobeIcon class="w-20 h-20 fill-like-green" />
           </ButtonV2>
         </template>
-        <Menu
-          :items="availableLocales"
-          :selected-value="currentLocale"
-          @select="handleSelectLocale"
-        />
+        <MenuList>
+          <MenuItem
+            :items="availableLocales"
+            :selected-value="currentLocale"
+            @select="handleSelectLocale"
+          />
```

Comment:
I suggest to for-loop `<MenuItem>` instead of using the `items` prop, more flexibility for customization